vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>Why not put this in a dictionary as in:</p>

<pre class="ql-syntax" spellcheck="false">segmentation_classes_dict = {
    "backpack": 0,
    ...
}
</pre><p> </p><p>This way, users can find the full list by using:</p>

<pre class="ql-syntax" spellcheck="false">segmentation_classes_dict.keys()
</pre><p> </p><p>And get the ID directly based on the instance type, if it exists:</p>

<pre class="ql-syntax" spellcheck="false">segmentation_classes_dict.get("backpack")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
This one I feel might be a little overkill. I'm imagining a user who wants to set a segmentation class will have an easier time just scrolling down to the table and CTRL+F looking for the desired class.